articles/app-service-web/app-service-linux-intro.md

```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 ms.author: naziml;wesmc
 
 ---
-# Introduction to Azure Web App on Linux
+# Introduction to Azure Web Apps on Linux
 
 [!INCLUDE [app-service-linux-preview](../../includes/app-service-linux-preview.md)]
 
 ## Overview
-Customers can use Web App on Linux to host web apps natively on Linux for supported application stacks. The following section lists the application stacks that are currently supported. 
-
-## Features
-Web App on Linux currently supports the following application stacks:
-
-* Node.js
-	* 4.4
-	* 4.5
-	* 6.2
-	* 6.6
-	* 6.9
-	* 6.10
-	* 6.11
-	* 8.0
-	* 8.1
-* PHP
-	* 5.6
-	* 7.0
-* .Net Core
-	* 1.0
-    * 1.1
-* Ruby
-	* 2.3
-
-Customers can deploy their applications by using:
+[Web Apps](https://docs.microsoft.com/en-us/azure/app-service-web/app-service-web-overview) is a fully managed compute platform that is optimized for hosting websites and web applications. Customers can use Web Apps on Linux to host web apps natively on Linux for supported application stacks. The following sections lists the application stacks that are currently supported. 
+
+### Languages
+
+|       Node.js      | PHP | .NET Core | Ruby |
+|:------------------:|:---:|:---------:|:----:|
+|      4.4, 4.5      | 5.6 |  1.0-1.1  |  2.3 |
+| 6.2, 6.6, 6.9-6.11 | 7.0 |           |      |
+|       8.0-8.1      |     |           |      |
+
+### Deployments
 
 * FTP
 * Local Git
 * GitHub
 * Bitbucket
 
-For application scaling:
+### DevOps
 
-* Customers can scale web apps up and down by changing the tier of their App Service plan
-* Customers can scale out applications and run multiple app instances within the confines of their SKU
+* Staging environments
+* ACR and DockerHub CI/CD
 
-For Kudu, some of the basic functionality:
+### Console, Publishing, and Debugging
 
 * Environments
 * Deployments
 * Basic console
 * SSH
 
-For devops:
+### Scaling
 
-* Staging environments
-* ACR and DockerHub CI/CD
+* Customers can scale web apps up and down by changing the tier of their [App Service plan](https://docs.microsoft.com/en-us/azure/app-service/azure-web-sites-web-hosting-plans-in-depth-overview?toc=%2fazure%2fapp-service-web%2ftoc.json)
 
 ## Limitations
-The Azure portal shows only features that currently work for Web App on Linux and hides the rest. As we enable more features, they will be visible on the portal.
+The Azure portal shows only features that currently work for Web Apps on Linux. As we enable more features, they will become visible on the portal.
 
 Some features, such as virtual network integration, Azure Active Directory/third-party authentication, or Kudu site extensions, are not available yet. Once these features are available, we will update our documentation and blog about the changes.
 
 This public preview is currently only available in the following regions:

```

Comment:
It would be helpful to provide a link to the [Azure Status Dashboard](https://azure.microsoft.com/status) to let people know which regions will be available.